go.mod

```diff
@@ -17,8 +17,8 @@ go 1.21
 // github.com/launchdarkly/go-server-sdk/v7 back to "latest" and commit the change.
 
 require (
-	github.com/launchdarkly/go-sdk-common/v3 latest
-	github.com/launchdarkly/go-server-sdk/v7 latest
+	github.com/launchdarkly/go-sdk-common/v3 v3.1.0
```

Comment:
Do we still want to keep this `latest` @keelerm84 @cwaldren-ld? This was what "encouraged" people to pick the actual latest version of the SDK.

Reply:
Yes, that is my mistake. Restored!

Reply:
Now that this is in GHA and terraform, we can switch to Dependabot for updating this